frontend/src/common/components/naviconsidebar/NavIconReportButton.tsx

```diff
@@ -0,0 +1,27 @@
+import './NavIconReportButton.scss'
+import { IconButton, Tooltip } from "@mui/material";
+import { Note } from "@mui/icons-material";
+import { useNavigate } from "react-router-dom";
+import * as routes from "../../../routes/constants";
+
+/**
+ * Displays the navigation icon for Reports on the NavIconSideBar
+ */
+export const NavIconReportButton = () => {
+    
+    const navigate = useNavigate()
+
+    return (
+        <div className="nav-icon-report-button-container">
+            <Tooltip arrow placement="left" title="Reports">
+                <IconButton
+                    size="medium"
+                    color="secondary"
+                    onClick={() => {navigate(routes.PERMITS)}}
```

Comment:
we don't seem to have a "reports" type of route, so I went with `PERMITS` 🤷

Reply:
Right, currently we don't have the reports route, but I think Krishnan's working on the reports stuff so I guess we'll have to wait until his changes gets into main. But for the time being, I think this should navigate to routes.IDIR_WELCOME as well, and we can change it to point to the reports route once Krishnan's changes get merged.

Reply:
This is because not all IDIR users have the role to access the PERMITS route, if any. So we can use IDIR_WELCOME as a default placeholder route for the time being.

Reply:
sounds good - updated